### PR TITLE
[d16-5-xcode11.4] Merge xcode11.4 into d16-5

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

**Comment by dalexsoto** [comment]
Known failure: HE0038

File: tools/linker/RemoveRejectedTypesStep.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+		public DerivedLinkContext Context {
+			get { return context as DerivedLinkContext; }
+		}
+
+		public override SubStepTargets Targets {
+			get { return SubStepTargets.Assembly; }
+		}
+
+		protected override string Name { get; } = " Removing Rejected Type";
```
Comment (whitneyschmidt) [inline tools/linker/RemoveRejectedTypesStep.cs]: Extra space at beginning of `Removing Rejected Type`?

Comment (mandel-macaque) [reply tools/linker/RemoveRejectedTypesStep.cs]: To late for this, since is a cherry-pick/backport. We should fix that in the correct location :/

Comment (spouliot) [reply tools/linker/RemoveRejectedTypesStep.cs]: Good catch :-)
It's only shown on on errors, obviously we hope never to throw unexpectedly here, and won't affect execution (nor would it be a breaking change)

**Pull request merged**